### ISSUE TITLE
Sound not working with iOS 11.4.1 + Safari 11.0 Mobile + iOS 9.3.5

### ISSUE THREAD

**Issue by lmulhall-phet** [phetsims/vibe#35]
Found while doing testing for https://github.com/phetsims/special-ops/issues/112#issuecomment-409089475.

This is a bizarre issue. Sound in our simulations is not working with iOS 11.4.1 + Safari 11.0 Mobile. Sound isn't working at all on Pauling (iPad Air 2 + iOS 11.4.1 + A1566), but does seem to work initially on Dalton (iPad Air 2 + iOS 11.4.1 + A1566). However, if you leave the sim running for a bit on Dalton -- roughly 2 minutes -- the sound stops working. I am seeing Dalton's behavior on my iPad Pro (iOS 11.4.1 + A1670).

Also, the sound seems to work with headphones, but this is what it sounds like...

Correct Answer Sound:
https://drive.google.com/file/d/1R7ObGHbz3jWFPg2hixTKMSciWkPgnc5v/view?usp=sharing

Wrong Answer Sound:
https://drive.google.com/open?id=1as39hOWa9nFxVn_m7QWx8xIXyZUwF92Z

Before you ask, no, this is not due to the sound being recorded through headphones; this is what it actually sounds like.

I can't really tell if this is an issue with iOS 11 or Mobile Safari. I don't think it's an issue we introduced because sound isn't working in published sims. ¯\\_(ツ)_/¯

Sound is working on iOS 9 devices.

I have no idea if this is helpful, but there is a note in [About the security content of iOS 11.4.1](https://support.apple.com/en-us/HT208938)...
![screen shot 2018-08-01 at 11 58 49 am](https://user-images.githubusercontent.com/31285590/43539757-0357017e-9583-11e8-9c07-bac39ff75dfb.png)

EDIT: While doing testing for https://github.com/phetsims/QA/issues/160, I noticed that leaving https://phet-dev.colorado.edu/html/tambo/1.0.0-dev.4/phet/tambo_en_phet.html running for a while on Tycho (iPad 2 + iOS 9.3.5) makes the sound not work. This leads me to believe that this is not a problem specific to iOS version or Mobile Safari version.

**Comment by KatieWoe** [comment]
This issue can occur after leaving phetsims/QA/issues/162 running for several minutes on Tycho (iPad 2 + iOS 9.3.5).  This lends support to @lmulhall-phet s thought that this is not necessarily iOS 11.4.1 exclusive.
@pixelzoom @jonathanolson

**Comment by pixelzoom** [comment]
@KatieWoe Is this problem also present in published versions that preceded @jbphet's recent changes to sound?

**Comment by pixelzoom** [comment]
For example... Is this problem present in [Equality Explorer 1.0.0-rc.2](https://bayes.colorado.edu/dev/html/equality-explorer/1.0.0-rc.2/phet/equality-explorer_en_phet.html)?  If not, then the problem was likely introduced by @jbphet's recent sound changes. The branch he was working in was merged to master _after_ the publication of Equality Explorer 1.0.0.

**Comment by KatieWoe** [comment]
@pixelzoom 
Just tested this issue on https://bayes.colorado.edu/dev/html/equality-explorer/1.0.0-rc.2/phet/equality-explorer_en_phet.html with an iOS 11.4.1 device and the sound is not playing at all.  Have not yet encountered the iOS 9 version of the problem where the sound cuts out after a certain amount of time.

**Comment by KatieWoe** [comment]
I will note that the iPad 2 already has a tendency to crash on some sims if they are left open for a time.  This is a known issue and only dealt with if the crashes happen very frequently.  When it crashes it reloads and this fixes any sound issues that occurred.  I did just find that after being open for awhile, the iOS 9 device has lost sound on the equality explorer 1.0.0-rc.2

**Comment by pixelzoom** [comment]
Lost sound + crashing probably equates to running out of memory.  And Equality Explorer is an especially large sim.

**Comment by KatieWoe** [comment]
I agree.  This makes me think that the iOS 9 and the iOS 11 issues may be separate after all.

**Comment by pixelzoom** [comment]
@ariel-phet This issue is blocking RC testing of RPAL, https://github.com/phetsims/QA/issues/162.  Do you want to wait until it is addressed, or proceed with RC testing and address in a future maintenance patch?

**Comment by ariel-phet** [comment]
@pixelzoom since this issue apparently exists in the published sim (and has not been user reported) I think it is fine to address in a future maintenance patch.

**Comment by KatieWoe** [comment]
As noted in https://github.com/phetsims/vibe/issues/35#ref-issue-352589854, this issue also seems to be present in the app, meaning it may not necessarily be a safari based issue.
@jbphet @jonathanolson @pixelzoom

**Comment by KatieWoe** [comment]
I've been playing around with fixes I've found for 11.4.1 sound issues, and it seems that one of them has worked for Pauling.  I'll keep poking at it for the next few hours to see if the issue comes back and keep you updated.

**Comment by KatieWoe** [comment]
Documenting some of the steps used in fix here.  I didn't notice it was fixed for some time, so unfortunately I'm not sure what ultimately did it.
Resetting
Turning off bluetooth
settings>sounds>turn on/off/on "Change with Buttons">Alter volume
Adjust notification setting
Turn on/off/on ringer (bell icon)

**Comment by jbphet** [comment]
I just tested a published sim (Area Model Algebra) on Pauling (iOS version 11.4.1) and Tycho (iOS version 9.3.5) and sound generation worked fine on both.  I also tested an [RC of the John Travoltage sim](https://phet-dev.colorado.edu/html/john-travoltage/1.5.0-rc.1/phet/john-travoltage_en_phet.html), and that also worked fine.  I discussed this with @KatieWoe, and we decided to assign it to @oliver-phet so that he is aware of the potential workaround described in the previous comment, and can let us know if we get any complaints from users about the sound behavior.  @oliver-phet - after you've looked through this, feel free to close it.
